internal/gps/vcs_repo.go

```diff
@@ -92,6 +92,8 @@ func (r *gitRepo) get(ctx context.Context) error {
 		r.Remote(),
 		r.LocalPath(),
 	)
+	// Ensure no prompting for PWs
+	cmd.SetEnv(append([]string{"GIT_ASKPASS=", "GIT_TERMINAL_PROMPT=0"}, os.Environ()...))
```

Comment:
When dep runs git commands, does it inherit our current environment variables? I'm wondering if we can just set this once?

Reply:
I'm not sure - I copied from this: https://github.com/golang/dep/blob/80e94e8eef3660a2cd9d80b64ae95d7c898e8453/gps/vcs_source.go#L241-L246

Reply:
Ah gotcha, yeah it doesn't inherit any environment variables then. 👍